### DISCUSSION TITLE
Non Caster Weapon Support

### DISCUSSION THREAD

**Discussion by livejamie** [xanthics/PoE_Weighted_Search#32] [Ideas]
Would be nice to support weapons other than caster, like bows, claws, etc.

**Comment by xanthics** [comment]
Unfortunately you cannot model attack weapons with weights.  The main trade site doesn't support local stats.  So something having 15% attack speed, 35% crit, 120% increased physical damage, and ~24ish flat would look very different on various weapon bases.

**Comment by livejamie** [comment]
Ah that's a bummer, thank you for the response.

Btw this tool is invaluable thank you so much for maintaining it. I use it every league and it saves me a lot of time and money.
